public/styles/commonclient.css

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Please rename this file to `commonClient.css` for consistency with my camelCase naming scheme.